src/app/EventManagement.cpp

```diff
@@ -97,6 +97,7 @@ struct EventEnvelopeContext
     EndpointId mEndpointId   = 0;
     EventId mEventId         = 0;
     EventNumber mEventNumber = 0;
+    Optional<FabricIndex> mFabricIndex;
```

Comment:
Again, document.  Maybe by just pointing to the documentation for EventOptions?

Reply:
done, thanks